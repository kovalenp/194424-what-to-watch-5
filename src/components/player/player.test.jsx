import React from "react";
import renderer from "react-test-renderer";
import Player from "./player";
import movie from "../../mocks/movie";

describe(`Player meets B21`, () => {

  it(`and rendered correctly for auth'ed`, () => {
    const tree = renderer.create((
      <Player
        movie={movie} />
    )).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
