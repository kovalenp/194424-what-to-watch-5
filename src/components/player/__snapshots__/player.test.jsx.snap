// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player meets B21 and rendered correctly for auth'ed 1`] = `
<div
  className="player"
  style={
    Object {
      "backgroundColor": "#000000",
    }
  }
>
  <video
    controls={true}
    height="100%"
    muted={false}
    poster="https://assets.htmlacademy.ru/intensives/javascript-3/film/background/bronson.jpg"
    preload="none"
    src="http://media.xiph.org/mango/tears_of_steel_1080p.webm"
    width="100%"
  />
  <button
    className="player__exit"
    onClick={[Function]}
    style={
      Object {
        "top": "12px",
      }
    }
    type="button"
  >
    Exit
  </button>
</div>
`;
